.travis.yml

```diff
@@ -7,11 +7,14 @@ install:
 script:
   - tox -e py27
   - (cd core && tox -e py27)
+  - (cd datastore && tox -e py27)
   - tox -e py34
   - (cd core && tox -e py34)
+  - (cd datastore && tox -e py34)
   - tox -e lint
   - tox -e cover
   - (cd core && tox -e cover)
+  - (cd datastore && tox -e cover)
```

Comment:
I wonder what the build times will be once all the packages are split if we do this.

I know there is a desire, but do you have a plan on how to only build the modified packages?